pkg/database/realm.go

```diff
@@ -42,6 +48,19 @@ type Realm struct {
 	// Name is the name of the realm.
 	Name string `gorm:"type:varchar(200);unique_index"`
 
+	// Code configuration
+	RegionCode              string `gorm:"type:varchar(5); not null; default: ''"`
+	CodeLength              uint   `gorm:"type:smallint; not null; default: 8"`
+	CodeDurationSeconds     int64  `gorm:"type:bigint; not null; default: 900"` // default 15m (in seconds)
```

Comment:
The max for integer is 2147483647. 2147483647 is ~70 years. bigint is 9223372036854775807 which is 2924712086 _centuries_. I think we can use integer here 😄